### PR TITLE
add flavour cmd

### PR DESCRIPTION
**Pull request by tornado404**

### CODE REVIEW

File: client/paddleflow/cli/flavour.py [hunk 1]
```diff
@@ -51,10 +53,123 @@ def list(ctx):
...
+    else:
+        click.echo("get flavour failed with message[%s]" % response)
+        sys.exit(1)
+
+
+@flavour.command()
+@click.option('-n',"--flavourname", required=True)
+@click.option('-c', '--cpu', help="CPU")
+@click.option('-m', '--memory', help="Memory, e.g. --memroy 10G")
+@click.option('-s','--scalar', help='the scalar resource of flavour, e.g. --scalar a=b,c=d')
```
Comment (D0m021ng) [inline client/paddleflow/cli/flavour.py]: `the` => `The`

File: client/paddleflow/cli/flavour.py [hunk 2]
```diff
@@ -51,10 +53,123 @@ def list(ctx):
...
+                   (clustername, response))
+    else:
+        click.echo("cluster update failed with message[%s]" % response)
+        sys.exit(1)
+
+
+@flavour.command()
+@click.option('-n', "--flavourname", required=True)
+@click.option('-c', '--cpu', help="CPU", required=True)
+@click.option('-m', '--memory', help="Memory.", required=True)
```
Comment (D0m021ng) [inline client/paddleflow/cli/flavour.py]: The help information is too short. `The memory of flavour`?

File: client/paddleflow/cli/flavour.py (hunk 1 above)
Comment (tornado404) [reply client/paddleflow/cli/flavour.py]: done

File: client/paddleflow/cli/flavour.py (hunk 2 above)
Comment (tornado404) [reply client/paddleflow/cli/flavour.py]: ok

**Review by D0m021ng — APPROVED** [review]

**Pull request merged**